driver-core/src/main/java/com/datastax/driver/core/querybuilder/Insert.java

```diff
@@ -56,7 +56,7 @@ StringBuilder buildQueryString(List<Object> variables) {
         if (keyspace != null)
             Utils.appendName(keyspace, builder).append('.');
         Utils.appendName(table, builder);
-        builder.append('(');
+        builder.append(" (");
```

Comment:
Not required, but imho the resulting string looks better with this extra whitespace between the table name and the columns list.